src/main/resources/META-INF/plugin.xml

```diff
@@ -50,6 +50,7 @@
 
     <change-notes><![CDATA[
       <ul>
+        <li>4.8 - Support offline configuration of rule parameters. Performance improvement and fewer false positives for PHP. 4 new vulnerability detection rules for Python.</li>
```

Comment:
Is `offline` the new name for `standalone`? I don't find it very explicit. Maybe `local` instead? Or instead `Support configuration of rules parameters in IDE settings`. No need to emphasis this is not for connected mode IMO.